eth/scripts/2_setup_contracts.js

```diff
@@ -114,6 +114,8 @@ async function main() {
   await migrations.setCompleted(2);
 
   console.log("Done");
+  // NOTE: neccessary because script hangs in CI
+  process.exit(0);
```

Comment:
O.o

Reply:
if it's stupid but works it's not stupid :man_shrugging: